packages/flutter_tools/lib/src/runner/target_devices.dart

```diff
@@ -605,13 +616,15 @@ class TargetDevicesWithExtendedWirelessDeviceDiscovery extends TargetDevices {
   ) async {
     final Device? ephemeralDevice = _deviceManager.getSingleEphemeralDevice(attachedDevices);
     if (ephemeralDevice != null) {
+      stopExtendedWirelessDeviceDiscovery();
```

Comment:
Is this needed? `_handleRemainingDevices` is called within `findAllTargetDevices`, which calls `stopExtendedWirelessDeviceDiscovery`

Reply:
done